egs/public_dataset/ofuton_p_utagoe_db/conf/infer_puretransformer.yaml

```diff
@@ -0,0 +1,23 @@
+test_align: /export/c04/jiatong/project/svs/SVS_system/data/alignment/test/
```

Comment:
For files in `conf`, please refer to Kiritan and kept those major configurations.